tests/test_tztrout.py

```diff
@@ -0,0 +1,433 @@
+            ('US', 'IL', 'Chicago', ['+16303334444'], 'CT', True),
+            ('US', 'TX', 'Houston', ['+17133334444'], 'CT', True),
+            ('US', 'PA', 'Philadelphia', ['+12153334444'], 'ET', True),
+            ('US', 'AZ', 'Phoenix', ['+16023334444'], 'MT', True),
+            ('US', 'TX', 'San Antonio', ['+12103334444'], 'CT', True),
+            ('US', 'CA', 'San Diego', ['+16193334444'], 'PT', True),
+            ('US', 'TX', 'Dallas', ['+12143334444'], 'CT', True),
+            ('US', 'CA', 'San Jose', ['+14083334444'], 'PT', True),
+            ('US', 'TX', 'Austin', ['+15123334444'], 'CT', True),
+            ('US', 'IN', 'Indianapolis', ['+13173334444'], 'ET', True),
+            ('US', 'FL', 'Jacksonville', ['+19043334444'], 'ET', True),
+            ('US', 'CA', 'San Francisco', ['+14153334444'], 'PT', True),
+            ('US', 'OH', 'Columbus', ['+16143334444'], 'ET', True),
+            ('US', 'NC', 'Charlotte', ['+17043334444'], 'ET', True),
+            ('US', 'TX', 'Fort Worth', ['+16823334444'], 'CT', True),
+            ('US', 'MI', 'Detroit', ['+13133334444'], 'ET', True),
+            ('US', 'TX', 'El Paso', ['+19153334444'], 'MT', False),
+            ('US', 'TN', 'Memphis', ['+19013334444'], 'CT', False),
+            ('US', 'CO', 'Denver', ['+13033334444'], 'MT', True),
+            ('US', 'DC', 'Washington', ['+12023334444'], 'ET', False),
+            # Canada
+            ('CA', 'ON', 'Toronto', ['+14163334444'], 'ET', False),
+            ('CA', 'QC', 'Montreal', ['+15143334444'], 'ET', False),
+            ('CA', 'AB', 'Calgary', ['+14033334444'], 'MT', False),
+            (
+                'CA',
+                'ON',
+                'Ottawa',
+                ['+13433334444', '+16133334444'],
+                'ET',
+                False,
+            ),
+            ('CA', 'AB', 'Edmonton', ['+17803334444'], 'MT', False),
+            ('CA', 'ON', 'Mississauga', ['+12893334444'], 'ET', False),
+            ('CA', 'MB', 'Winnipeg', ['+14313334444'], 'CT', False),
+            ('CA', 'BC', 'Vancouver', ['+16043334444'], 'PT', False),
+            ('CA', 'NS', 'Halifax', ['+19023334444'], 'AT', False),
+            ('CA', 'SK', 'Saskatoon', ['+13063334444'], 'CT', False),
+        ],
+    )
+    def test_major_cities_us_ca(
+        self,
+        country,
+        state,
+        city,
+        phones,
+        tz_name,
+        assert_ids_equal,
+        us_ca_tz_names,
+    ):
+        """ Make sure all the major cities in the United States and Canada match the right time zone
+        (and the right time zone only). """
+        ids = tztrout.tz_ids_for_address(country, state=state, city=city)
+        for phone in phones:
+            ids2 = tztrout.tz_ids_for_phone(phone)
+            if assert_ids_equal:
+                assert ids == ids2
+            else:
+                self.assert_only_one_tz(ids2, tz_name, us_ca_tz_names)
+        self.assert_only_one_tz(ids, tz_name, us_ca_tz_names)
```

Comment:
This test seems to be testing two things (`tz_ids_for_address` and `tz_ids_for_phone`). They should both be tested separately.

Reply:
Done in 20a80d7